samples/client/petstore/typescript-axios/builds/with-npm-version/package.json

```diff
@@ -22,7 +22,7 @@
     "prepare": "npm run build"
   },
   "dependencies": {
-    "axios": "^0.26.1"
+    "axios": "^1.2.3"
```

Comment:
update to 1.4.0 if possible [1.4.0](https://www.npmjs.com/package/axios/v/1.4.0)

Reply:
Is it possible to update to 1.6.0. Previous versions contain CSRF vulnerability

Reply:
https://github.com/OpenAPITools/openapi-generator/pull/16977

Reply:
@apptio-mrejdych / @MajdT51 can this be used then? https://github.com/OpenAPITools/openapi-generator/pull/16977